.cargo/config.toml

```diff
@@ -6,6 +6,7 @@ codegen-bindings      = "run -p xtask_codegen --features schema -- bindings"
 codegen-configuration = "run -p xtask_codegen --features configuration -- configuration"
 # updates a newly created crate
 codegen-license = "run -p xtask_codegen --features license -- license"
+codegen-migrate = "run -p xtask_codegen --features configuration -- migrate-eslint"
```

Comment:
We should add this command to the `justfile` too. Maybe to the `gen-all` and `gen-lint` commands